app/domain/authentication/update_authenticator_config.rb

```diff
@@ -0,0 +1,60 @@
+require 'command_class'
+
+module Authentication
+
+  UpdateAuthenticatorConfig = CommandClass.new(
+    dependencies: {
+      webservice_class: ::Authentication::Webservice,
+      resource_class: ::Resource,
+      authenticator_config_class: ::AuthenticatorConfig
+    },
+    inputs: %i(account authenticator service_id enabled current_user)
+  ) do
+    
+    def call
```

Comment:
what happens if you send an `update` request with an account that doesn't exist? if it fails with a misleading message (e.g `resource some-non-existing-account:webservice:conjur/authn-blah`) please use [ValidateAccountExists](https://github.com/cyberark/conjur/blob/master/app/domain/authentication/validate_account_exists.rb).

 You can see a usage [here](https://github.com/cyberark/conjur/blob/aac94ad69d2d13e07ac9a90e5e9288e01121139b/app/domain/authentication/validate_webservice_access.rb#L32)

Also, after the fix (if needed) please add tests for this. You can see an example [here](https://github.com/cyberark/conjur/blob/aac94ad69d2d13e07ac9a90e5e9288e01121139b/cucumber/authenticators/features/authn_oidc.feature#L131)

Reply:
Added validation using `ValidateAccountExists`.